packages/vm/package.json

```diff
@@ -64,6 +64,7 @@
   "devDependencies": {
     "@types/benchmark": "^1.0.33",
     "@types/core-js": "^2.5.0",
+    "@types/functional-red-black-tree": "^1.0.1",
```

Comment:
Couldn't 100% decide if to put in dependencies or dev dependencies, comments appreciated!

Reply:
I thought types always went in devDependencies.  That's where I'd lean.

Reply:
It's a bit more differentiated. If you are exporting the type in some way, e.g. if it is used for a parameter as an input type or if a return value on the API is returning a value with the respective type, the types package should be in the main dependencies, see first answer from [here](https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies) for a good summary.

In this specific case I wondered if people who fuddle around with their own StateManager implementation (and e.g. eventually also accessing the cache in some way) would benefit from a direct dependency on this.

Not sure, but eventually overblow. So for now here I decided to put into dev dependencies.